src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,89 @@
+/////////////////////////////////////////////////////////
+//
+//  Class: StringManipulation
+//  Author: Caleb Rector
+//  Date: 6/11/2023
+// 
+//  Description: Implements methods in StringManipulationInterface
+//
+/////////////////////////////////////////////////////////
+
 public class StringManipulation implements StringManipulationInterface {
 
+    private String s;
+
     @Override
     public String getString() {
-        return null;
+        return s;
     }
 
     @Override
     public void setString(String string) {
+        s = string;
     }
 
     @Override
     public int count() {
-        return 0;
+        if(s.length() == 0)
+            return 0;
+
+        String[] arr = s.split("[\\s\\n]");
```

Comment:
This can be simplified as: String[] arr = s.split("\\s+")

There should be a '+' in case there are multiple spaces in between words